compiler/damlc/daml-stdlib-src/DA/Internal/Interface.daml

```diff
@@ -0,0 +1,49 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | MOVE Prelude interface functionality
+module DA.Internal.Interface (
+  HasInterfaceTypeRep(..),
+  HasToInterface(..),
+  HasFromInterface(..),
+  Implements,
+  toInterface,
+  toInterfaceContractId,
+  fromInterfaceContractId,
+) where
+
+import DA.Internal.Prelude
+import DA.Internal.Template.Functions
+import DA.Internal.LF
+
+class HasInterfaceTypeRep i where
+  interfaceTypeRep : i -> TypeRep
+
+class HasToInterface t i where
+  toInterface_ : t -> i
```

Comment:
We usually put the underscore before, not after, for functions that we don't want people to call directly. (See `HasTemplateTypeRep`.) I think it's better to stick to that convention, and call this method `_toInterface`.

Reply:
oh! thanks for pointing this out, I'll fix it

Reply:
done now!